src/App.jsx

```diff
@@ -11,6 +12,9 @@ const App = () => {
       <h4>h4</h4>
       <p>p</p>
       <a href="#/">a</a>
+      <Pattern color="midblue" size="M" />{' '}
```

Comment:
Remove the component from App. We're using Storybook to view our components so don't need this here.